src/main/groovy/com/entagen/jenkins/JenkinsApi.groovy

```diff
@@ -224,7 +225,8 @@ class JenkinsApi {
         http.handler.failure = { resp ->
             def msg = "Unexpected failure on $jenkinsServerUrl$path: ${resp.statusLine} ${resp.status}"
             status = resp.statusLine.statusCode
-            throw new Exception(msg)
+            println msg
+            this.success = false
```

Comment:
You are still overriding all posts to never throw an exception they fail. I still don't think this is the best approach.